tasks/speech-synthesis-and-audio-analysis/MLproject

```diff
@@ -0,0 +1,13 @@
+name: mlflow_MLproject_file
```

Comment:
@slavakurilyak , could you please suggest a category for this task? should it be fit within tasks/natural-language-processing or should we add a new category tasks/natural-language-understanding/text-to-speech? Any suggestion would help.